supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -590,6 +590,17 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[softirq]]
+==== image:images/yes.png[yes] softirq (noun)
+*Description*: A _software interrupt request (softirq)_ is a deferrable kernel routine that performs the required actions in response to an interrupt. For example, softirqs clear the receive ring buffer of a network adapter after the reception of one or more packets.
+
+*Use it*: yes
+
+*Incorrect forms*: SoftIRQ, Softirq (unless at the start of a sentence)
```

Comment:
Can you remove "(unless at the start of a sentence)"? Any caveats should be listed in the description section, and this is one that should be understood of all terms.

Reply:
I moved this info to the description.